### ISSUE TITLE
intermittent 500 errors for 2e-revise-workflow in prod

### ISSUE THREAD

**Issue by grpandurangipwc** [google/exposure-notifications-verification-server#1816]
### TL;DR
We are seeing intermittent 500/unknown errors for e2e-revise-workflow in production.
Details:   
 "error": "invalid character '<' looking for beginning of value",
    "caller": "e2e-runner/main.go:213",

**Expected behavior**
Successful execution of 
https://e2e-runner-xxxxxxxxxxxxxx-xxc.a.run.app/revise

**Observed behavior**
https://e2e-runner-xxxxxxxxxxxxxx-xxc.a.run.app/revise failed 
Errored for github.com/google/exposure-notifications-verification-server/cmd/e2e-runner/main.go

### Reproduction
Unable to reproduce. This step fails intermittently, with frequency : once in 2 days

**Screenshots**

**Environment**

- OS:
- Browser:
- Version:

**Additional information**

**Comment by sethvargo** [comment]
Are there any error logs for any services around that same time? Does the runner succeed on a second try?

**Comment by grpandurangipwc** [comment]
Responded on email with screenshot.

**Comment by sethvargo** [comment]
Thanks, as I said in the email, if the e2e-runner succeeded on a second try, you can ignore this error. We believe it is fixed in the next release (v0.22.0) which was just tagged.
